go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/shirleyleu/xk6-output-timescaledb
+
+go 1.16
+
+require (
+	github.com/jackc/pgtype v1.8.1
+	github.com/jackc/pgx/v4 v4.13.0
+	github.com/sirupsen/logrus v1.8.1
+	go.k6.io/k6 v0.33.0
+	gopkg.in/guregu/null.v3 v3.5.0 // indirect
```

Comment:
For the latest master k6 version, it's represented as a specific hash in the `go.mod`, right? There is not a better way to do this? 

`go.k6.io/k6 v0.33.1-0.20210803135148-b705acb12bc4`

Reply:
yes. Although I would just run `go mod edit -require  go.k6.io/k6@master && go mod tidy` instead of touching the go.mod by hand

Reply:
(I guess this is starting to get pretty unrelated:
Alright so the first part adds `go.k6.io/k6 master` but then the `go mod tidy` adds the logrus and null.v3 lines which I have to delete by hand.... (?))

Reply:
Hmmm ... I guess this is because you actually use them in the code so they need to be in go.mod :(, sorry about that.

 Then, just run `go mod tidy` and commit with w/e it pushes in, it still should reduce the null.v3 version to 3.3.0 from my experiments.

Reply:
No worries! Yep it's v3.3.0 now

Does that mean I need manually update the master commit with `go mod edit -require go.k6.io/k6@master` from time to time? 😨

Reply:
not really, just after v0.33.0 we dropped somethings and it's better to use a version after it ;)